src/firefly/java/edu/caltech/ipac/firefly/server/visualize/VisServerOps.java

```diff
@@ -1544,6 +1545,8 @@ private static double getFluxFromFitsFile(File f,
                 if (clientFitsHeader ==null) {
                     throw new IOException("Can't read file, ClientFitsHeader is null");
                 }
+                //If test needed to debug,uncomment line below to test tpv for x,y IRSA-1009
+                //testTpv(f,ipt);
```

Comment:
Alternatively, you can do:
`if (ServerContext.DEBUG_MODE) testTpv(f,ipt);`
That way, method testTpv doesn't need to be commented out.  In debugger, you can call it even when DEBUG_MODE is set to false.

Reply:
Nice tip!